src/NCalc/ExpressionOptions.cs

```diff
@@ -63,15 +63,4 @@ public enum ExpressionOptions
     /// Allow calculation with boolean values
     /// </summary>
     AllowBooleanCalculation = 1 << 9,
-}
-
-public static class ExpressionOptionsExtensions
```

Comment:
To avoid a new breaking change, can we mark it as Obsolete until the next minor update?

Reply:
Very hard to someone use this outside NCalc, but good idea, not costs for us doing this.